test/SerialIOTest.cpp

```diff
@@ -4427,17 +4427,23 @@ TEST_CASE( "automatically_deactivate_span", "[serial][adios2]" )
     }
 
     // enable span-based API indiscriminately
-    try
     {
         std::string enable = R"END(
 {
   "adios2": {
     "use_span_based_put": true
   }
 })END";
-        Series write( "../samples/span_based.bp", Access::CREATE, enable );
-        auto E_uncompressed = write.iterations[ 0 ].meshes[ "E" ][ "x" ];
-        auto E_compressed = write.iterations[ 0 ].meshes[ "E" ][ "y" ];
+        /*
+         * Use a pointer so we can leak the memory later.
+         * We will leave write in an invalid state and the destructor results
+         * in a segfault.
+         * This is probably a bug in itself and should be fixed.
```

Comment:
Is this a bug in our lib or in ADIOS2?

Reply:
Probably on our side, I'm currently trying to figure it out.

Reply:
Fixed it now, see the commit message for details

Reply:
Ah perfect, thanks! Now we should remove this comment block again :)